src/items/AppItem.js

```diff
@@ -122,6 +131,20 @@ const AppItem = ({ id, item, user, apiHost }) => {
       />
     </React.Fragment>
   );
+
+  if (showCaption) {
```

Comment:
Just to confirm, we agreed that the caption always goes below, right? Otherwise we have to add something to switch it from below to above.

Reply:
Exactly, but this setting does not exist yet. We decided with André to have this display setting in `item.setting`. So I propose to have this feature in another PR.